magefile.go

```diff
@@ -127,9 +127,9 @@ func UpdateLibs() error {
 	return nil
 }
 
-// E2e runs e2e tests with a built plugin. Requires docker-compose.
+// E2e runs e2e tests with a built plugin against the example deployment. Requires docker-compose.
 func E2e() error {
-	return sh.RunV("docker-compose", "--file", "e2e/docker-compose.yml", "up", "--abort-on-container-exit")
+	return sh.RunV("docker-compose", "-f", "e2e/docker-compose.yml", "up", "--abort-on-container-exit", "tests")
```

Comment:
Does this rely on the example (`example/docker-compose.yml`) to be up and running already? If that is the case I'd recommend you bootstrap services on both files but in any case the command should not require you to do a previous command. See https://docs.docker.com/compose/extends/

Reply:
Another option is of course to launch the example before the e2e

Reply:
`mage e2e` does not rely anymore on `example/docker-compose.yml`. I thought that a standalone and straightforward docker-compose was a cleaner alternative.
(Mainly had troubles with `extends` because of `services with 'depends_on' cannot be extended`).